app/services/feature_flag.rb

```diff
@@ -28,6 +28,7 @@ def feature
     [:interviews, 'Providers can filter applications by interviewing state', 'Despo Pentara'],
     [:restructured_work_history, 'Candidates use the new design for the Work History section', 'David Gisbey'],
     [:structured_reasons_for_rejection_on_rbd, 'Allows providers to give specific feedback for applications rejected by default', 'Aga Dufrat'],
+    [:make_offer, 'Activates the new make offer flow for providers', 'Despo Pentara'],
```

Comment:
`:updated_offer_flow` perhaps?

`:make_offer` sounds too much like the general ability to make offers

Reply:
Addressed